src/client/renderer/r_image.c

```diff
@@ -161,13 +161,18 @@ void R_Screenshot(r_view_t *view) {
 	switch (r_image_state.screenshot) {
 		case SCREENSHOT_NONE:
 			return;
-		case SCREENSHOT_VIEW:
-			surface = R_ReadFramebuffer(view->framebuffer);
+		case SCREENSHOT_VIEW: {
```

Comment:
`r_screenshot view` will take a screenshot of the 3D view (minus HUD, crosshair, etc).